sample.env

```diff
@@ -1,8 +1,16 @@
 DEBUG=True
-LOGIN_ALLOWED_REDIRECT_DOMAINS=test.example.com:3000
```

Comment:
change history of this file: https://github.com/mozilla/network-pulse-api/commits/master/sample.env